librocksdb_sys/crocksdb/c.cc

```diff
@@ -2071,6 +2096,11 @@ void crocksdb_options_set_level_compaction_dynamic_level_bytes(
   opt->rep.level_compaction_dynamic_level_bytes = v;
 }
 
+unsigned char crocksdb_options_get_level_compaction_dynamic_level_bytes(
+    const crocksdb_options_t* const options) {
```

Comment:
Why two consts?

Reply:
the pointer and the value that the pointer point to are both not expect to be modified

Reply:
The pointer will not be modified since it is passed by value.

Reply:
I know, just notice people the input parameter is read only, actually this is trivial

Reply:
I think it is not necessary, otherwise, you should put a const before every input parameter.